packages/nextra-theme-docs/src/contexts/config.tsx

```diff
@@ -69,6 +69,23 @@ export function ConfigProvider({
     document.body.classList.toggle('_overflow-hidden', menu)
   }, [menu])
 
+  useEffect(() => {
+    let resizeTimer: number
+
+    function addResizingClass() {
+      document.body.classList.add('resizing')
+      clearTimeout(resizeTimer)
+      resizeTimer = window.setTimeout(() => {
+        document.body.classList.remove('resizing')
+      }, 200)
```

Comment:
Should we add a condition for applying the resizing class? This would avoid repeatedly adding it during a resize by checking if a timer is already running.

```ts
if (!resizeTimer) {
  document.body.classList.add('resizing')
}
clearTimeout(resizeTimer)
resizeTimer = window.setTimeout(() => {
  resizeTimer = undefined
  document.body.classList.remove('resizing')
}, 200)
```

Reply:
Yes, and call clearTimeout only if value is truthy